js/build_map.js

```diff
@@ -29,7 +29,7 @@ var waterUseViz = {
   stateAbrvs: [], // created in extractNames()
   nationalData: {},
   stateData: {},
-  isNotEmbed: (window.location.pathname === "/")
+  isNotEmbed: !RegExp("html").test(window.location.pathname)
```

Comment:
I think this is an improvement but still not foolproof, because http://dev-owi.usgs.gov/vizlab/water-use-15/index.html is a link that produces the page that someone could reasonably choose. What about testing for the full embed page name?
```js
isNotEmbed: !RegExp("embed-water-use-15.html").test(window.location.pathname)
```

I'd also sorta prefer to see a variable called `isEmbed` rather than `isNotEmbed` throughout, but I can live with either.

Reply:
Oh, I guess the index.html one exists, but none of our clickable things gets to that. Yes, I thought about isEmbed, too rather than double negative but would've had to change the tests. I can do that.

Reply:
Yeah, it's a relatively low risk, but seems worth the small change if the test for the full page name "embed-water-use-15.html" works (i hope it will - does `.` being a wild card mess things up with `RegExp()`?)